src/stackset_roles/template.yml

```diff
@@ -155,6 +160,75 @@ Resources:
         - Key: "aws-cloudformation:logical-id"
           Value: rCloudFormationRole
 
+  rApiGatewayLoggingRole:
```

Comment:
I added the API Gateway and AppSync logging roles because there is only one of these per AWS account and this felt like a good template to pre-provisioning those roles for customers.